generators/gulp/templates/gulpfile.js

```diff
@@ -12,6 +12,7 @@ var coveralls = require('gulp-coveralls');
 <% } -%>
 <% if (babel) { -%>
 var babel = require('gulp-babel');
+require('babel-core/register');
```

Comment:
Maybe adding a comment explaining the the require is doing would help. I'm fine with just requiring it.